pkgs/top-level/all-packages.nix

```diff
@@ -673,6 +673,8 @@ with pkgs;
 
   enumer = callPackage ../tools/misc/enumer { };
 
+  errands = callPackage ../applications/misc/errands { };
```

Comment:
Migrate to `pkgs/by-name` hierarchy.

Quick tuto: move `default.nix` to `pkgs/by-name/er/errands/package.nix` and remove this line:

```suggestion
```